bittensor/extrinsics/staking.py

```diff
@@ -523,3 +528,208 @@ def __do_add_stake_single(
     )
 
     return success
+
+
+def do_set_child_singular_extrinsic(
+    subtensor: "bittensor.subtensor",
+    wallet: "bittensor.wallet",
+    hotkey: str,
+    child: str,
+    netuid: int,
+    proportion: float,
+    wait_for_inclusion: bool = True,
+    wait_for_finalization: bool = False,
+    prompt: bool = False,
+) -> Tuple[bool, str]:
+    r"""
+    Sets child hotkey with a proportion assigned from the parent.
+
+    Args:
```

Comment:
Pls use the same line for argument and explanation. This will help avoid double work when generating documentation.

Instead of 
```python
"""
Sets child hotkey with...
...
        subtensor (bittensor.subtensor):
            Subtensor endpoint to use.
...
"""
```
use 
```python
"""
Sets child hotkey with...
...

        subtensor (bittensor.subtensor): Subtensor endpoint to use.
...
"""
```

The same in other docstrings.